db/migrate/20231129042832_create_comments.rb

```diff
@@ -1,9 +1,9 @@
 class CreateComments < ActiveRecord::Migration[7.1]
   def change
     create_table :comments do |t|
-      t.references :user, foreign_key: true
-      t.references :post, foreign_key: true
-      t.string :text
+      t.references :author, foreign_key: { to_table: :users }
+      t.references :post, foreign_key: { to_table: :users }
+      t.text :text
```

Comment:
- [ ] In the posts references, make the to_table: :posts and not users. 👍

Reply:
Hi@MahmoudBakr23. All issues are fixed as required.